Parent/Parent/Browser/WebBrowserViewController.swift

```diff
@@ -153,7 +153,7 @@ class WebBrowserViewController: UIViewController {
 
     @objc func reloadButtonTapped(_ refreshButton: UIBarButtonItem) {
         networkOps = 0
-        if let request = webView.request, (request.url?.absoluteString.characters.count)! > 0 && request.url?.absoluteString != "about:blank" {
+        if let request = webView.request, request.url?.absoluteString.isEmpty == false && request.url?.absoluteString != "about:blank" {
```

Comment:
`.characters` is deprecated. It was silly to use it here anyway.

Reply:
Better in so many ways.